Sources/TuistKit/Commands/GenerateCommand.swift

```diff
@@ -19,24 +19,52 @@ struct GenerateCommand: ParsableCommand, HasTrackableParameters {
     )
     var path: String?
 
+    @Argument(help: """
+    A list of targets in which you'd like to focus. \
+    Those and their dependant targets will be generated as sources. \
+    If no target is specified, the project defined targets will be focused.
+    """)
+    var sources: [String] = []
+
+    @Flag(
+        name: .shortAndLong,
+        help: "Don't open the project after generating it."
+    )
+    var noOpen: Bool = false
+
     @Flag(
+        name: [.customShort("x"), .long],
+        help: "When passed it uses xcframeworks (simulator and device) from the cache instead of frameworks (only simulator)."
+    )
+    var xcframeworks: Bool = false
+
+    @Option(
         name: [.customShort("P"), .long],
-        help: "Only generate the local project (without generating its dependencies)."
+        help: "The name of the cache profile to be used when focusing on the target."
     )
-    var projectOnly: Bool = false
+    var profile: String?
 
     @Flag(
-        name: [.customShort("O"), .long],
-        help: "Open the project after generating it."
+        name: [.customLong("no-cache")],
+        help: "Ignore cached targets, and use their sources instead."
     )
-    var open: Bool = false
+    var ignoreCache: Bool = false
```

Comment:
Id simplify this to just `--no-cache` with that being the only way of doing this

Reply:
Isn't it already `--no-cache` given the `.customLong` above?

Reply:
Oh yeah i was confused by the custom name but i see it's to enable having a more standard variable name in code, looks good!